fastapi/openapi/docs.py

```diff
@@ -1,45 +1,40 @@
+from typing import Dict
+
 from starlette.responses import HTMLResponse
 
 
-def get_swagger_ui_html(*, openapi_url: str, title: str) -> HTMLResponse:
-    return HTMLResponse(
-        """
+def get_swagger_ui_html(
+    *, openapi_url: str, title: str, swagger_locations: Dict
+) -> HTMLResponse:
+    html = f"""
     <! doctype html>
     <html>
     <head>
-    <link type="text/css" rel="stylesheet" href="https://cdn.jsdelivr.net/npm/swagger-ui-dist@3/swagger-ui.css">
-    <link rel="shortcut icon" href="https://fastapi.tiangolo.com/img/favicon.png">
-    <title>
-    """
-        + title
-        + """
-    </title>
+    <link type="text/css" rel="stylesheet" href="{swagger_locations.get("css")}">
+    <link rel="shortcut icon" href="{swagger_locations.get("favicon")}">
+    <title>{title}</title>
     </head>
     <body>
     <div id="swagger-ui">
     </div>
-    <script src="https://cdn.jsdelivr.net/npm/swagger-ui-dist@3/swagger-ui-bundle.js"></script>
+    <script src="{swagger_locations.get("js")}"></script>
     <!-- `SwaggerUIBundle` is now available on the page -->
     <script>
-            
-    const ui = SwaggerUIBundle({
-        url: '"""
-        + openapi_url
-        + """',
+    const ui = SwaggerUIBundle({{
+        url: '{openapi_url}',
         dom_id: '#swagger-ui',
         presets: [
         SwaggerUIBundle.presets.apis,
         SwaggerUIBundle.SwaggerUIStandalonePreset
         ],
-        layout: "BaseLayout",
-        deepLinking: true
- 
-    })
+        layout: "BaseLayout"
+
+    }})
     </script>
     </body>
     </html>
     """
-    )
+    return HTMLResponse(html)
 
 
 def get_redoc_html(*, openapi_url: str, title: str) -> HTMLResponse:
```

Comment:
Help me change this to accept a `redoc_js_url`, and a `redoc_favicon_url`.

Both of them being `str`, with a default value of the current URLs.